sycl/test-e2e/Matrix/XMX8/joint_matrix_transposeC.cpp

```diff
@@ -6,12 +6,11 @@
 //
 //===----------------------------------------------------------------------===//
 // REQUIRES: matrix-xmx8
+// REQUIRES-INTEL-DRIVER: lin: 28267
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-// XFAIL:gpu
```

Comment:
transpose has not been added for DG2, so please keep XFAIL here.

Reply:
Are you sure? Because this test start to pass with the newer driver....

Reply:
OK, fallback was added for transpose C.